packages/web/app/src/components/ui/modals/create-organization.tsx

```diff
@@ -104,50 +104,64 @@ export const CreateOrganizationForm = () => {
     }
   }
 
+  return <CreateOrganizationFormContent form={form} onSubmit={() => onSubmit(form.getValues())} />;
```

Comment:
```suggestion
  return <CreateOrganizationFormContent form={form} onSubmit={onSubmit} />;
```

Reply:
the `form` may or may not be in sync with submitted values, we assume it is, but the callback (called by `form.handleSubmit`) might do weird things.